src/main/java/org/ehrbase/aqleditor/configuration/EhrConfig.java

```diff
@@ -0,0 +1,33 @@
+package org.ehrbase.aqleditor.configuration;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import lombok.RequiredArgsConstructor;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.ehrbase.aqleditor.properties.EhrProperties;
+import org.ehrbase.client.openehrclient.OpenEhrClientConfig;
+import org.ehrbase.client.openehrclient.defaultrestclient.DefaultRestClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@RequiredArgsConstructor
+public class EhrConfig {
+
+  private final EhrProperties ehrProperties;
+
+  @Bean
+  public DefaultRestClient createEhrRestClient() throws URISyntaxException {
+    CredentialsProvider provider = new BasicCredentialsProvider();
+    provider.setCredentials(
+        AuthScope.ANY,
+        new UsernamePasswordCredentials(
+            ehrProperties.getUsername(), ehrProperties.getPassword()));
+
+    return new DefaultRestClient(
```

Comment:
Way is nothing done with  CredentialsProvider provide?

Reply:
Added credentials to rest client